data/Hints/scrubs.json

```diff
@@ -32,4 +33,4 @@
     },
     "groups": [],
     "disabled": []
-}
+}
```

Comment:
The newline removed here should be added back.

Reply:
This is what I get for having Trent send it to me :)  I'll get that fixed

Reply:
I'm actually not sure what you mean here.   When compared to other presets it looks exactly the same